<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QF3FilterEditor</class>
 <widget class="QDialog" name="QF3FilterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>173</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Filter Editor</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Manufacturer:</string>
     </property>
     <property name="buddy">
      <cstring>edtManufacturer</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtManufacturer"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Type:</string>
     </property>
     <property name="buddy">
      <cstring>cmbType</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbType">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>band pass</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_bandpass.png</normaloff>:/libqf3widgets/filter_bandpass.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>long pass</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_longpass.png</normaloff>:/libqf3widgets/filter_longpass.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>short pass</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_shortpass.png</normaloff>:/libqf3widgets/filter_shortpass.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>notch filter</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_notch.png</normaloff>:/libqf3widgets/filter_notch.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>neutral density filter</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_neutraldensity.png</normaloff>:/libqf3widgets/filter_neutraldensity.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>polarizing</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_pol_lin.png</normaloff>:/libqf3widgets/filter_pol_lin.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dichroic splitter</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_dichroic.png</normaloff>:/libqf3widgets/filter_dichroic.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>neutral beam splitter</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_splitterneutral.png</normaloff>:/libqf3widgets/filter_splitterneutral.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>polarizing splitter</string>
      </property>
      <property name="icon">
       <iconset resource="libqf3widgets.qrc">
        <normaloff>:/libqf3widgets/filter_splitterpol.png</normaloff>:/libqf3widgets/filter_splitterpol.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>edtManufacturer</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="libqf3widgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QF3FilterEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QF3FilterEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
